crates/types/src/line.rs

```diff
@@ -95,10 +106,14 @@ impl<const DIMENSION: usize> Line<DIMENSION> {
         self.squared_distance_to_point(point).sqrt()
     }
 
-    pub fn is_orthogonal(&self, other: &Line<DIMENSION>, epsilon: f32) -> bool {
+    pub fn angle_diff_to_orthogonal(&self, other: &Line<DIMENSION>) -> f32 {
         let self_direction = (self.1 - self.0).normalize();
         let other_direction = (other.1 - other.0).normalize();
-        (self_direction.dot(&other_direction).acos().abs() - PI / 2.0).abs() < epsilon
+        (self_direction.dot(&other_direction).acos().abs() - PI / 2.0).abs()
```

Comment:
We use the premade constants pretty much everywhere else.
```suggestion
        (self_direction.dot(&other_direction).acos().abs() - FRAC_PI_2).abs()
```

Reply:
Yep, changed.